samples/VulkanInfo/src/main.cpp

```diff
@@ -212,6 +212,9 @@ int main(int argc, char *argv[])
         std::cout << "\t\t" << deviceExtensions[i].extensionName << " (Version " << deviceExtensions[i].specVersion << ")" << std::endl;
       }
     }
+
+    std::cout << std::endl << "Press enter to exit";
+    std::cin.ignore();

```

Comment:
I don't think this should stay in the codebase.

Reply:
Removed that.